ui/src/main/webapp/app/query/operation-chain-component.js

```diff
@@ -135,6 +135,66 @@ function OperationChainController(operationChain, config, loading, query, error,
         });
     }
 
+    vm.saveChain = function(ev) {
```

Comment:
This needs testing before it can be merged

Reply:
Do you want just jasmine tests? What about java tests in QueryBuilderTS.java?

Reply:
I meant Jasmine tests. However you could add a System test to be extra thorough